acceptancetests/assess_budget.py

```diff
@@ -241,12 +242,28 @@ def parse_args(argv):
     add_basic_testing_arguments(parser)
     return parser.parse_args(argv)
 
+def feed_cookie():
+    """Plant pre-generated cookie file to avoid launching Browser."""
+    user_home = os.getenv('HOME', default='/var/lib/jenkins')
+    juju_home = os.getenv('JUJU_HOME',
+                            default='/var/lib/jenkins/cloud-city')
+    ctrl_name = os.getenv('JOB_NAME', default='budget-management')
+    cookies_source = '{}/go-cookies-extended'.format(user_home)
+    cookies_target = '{}/jes-homes/{}/cookies/{}.json'.format(juju_home, ctrl_name, ctrl_name)
+    try:
+        subprocess.check_call(['cp', cookies_source, cookies_target])
+        log.info('Cookie file go-cookies-extended has been planted.')
+    except subprocess.CalledProcessError as e:
```

Comment:
JujuAssertError is only intended for when juju screws up (as opposed to the script, or a provider etc.)

Is it actually jujus fault if this call fails? If so that's fine, but if not raise a better suited expection.
Also, it would be ideal to have some reason why the call failed, perhaps include the exception string output as well.

Note, this won't pass flake8/pep8 as e is never used. That being said, I'm not sure the best way to run flake8 in this new world of being within the juju/juju repo. So lets tackle that problem later.